src/main/java/net/minecraftforge/fluids/IFluidTank.java

```diff
@@ -1,33 +1,16 @@
-/*
```

Comment:
Why was this removed?

Reply:
I believe KL copypasted the entire file from his draft implementation elsewhere, and that draft didn't have the license headers in it.